clients/cpp-console/scripts/linux/buildArmLinux.sh

```diff
@@ -45,13 +45,15 @@ fi
 
 echo "Building Linux Arm sample ..."
 if g++ -Wno-psabi \
+src/cppCore/cppCore.cpp \
```

Comment:
This file does nothing you shouldn't need it

Reply:
Done